test/functionalTest/cases/1492_limit_zero/limit_zero_regs.test

```diff
@@ -0,0 +1,267 @@
+# Copyright 2021 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+limit_zero_regs
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. GET /v2/registrations with 'limit' == 0 and options=count and see no entities and fiware-total-count: 0
+# 02. POST /v2/registrations to create E1 entity
+# 03. POST /v2/registrations to create E2 entity
+# 04. GET /v2/registrations with 'limit' == 0 and options=count and see no entities and fiware-total-count: 2
+# 05. GET /v2/registrations with 'limit' == 0 and see no entities
```

Comment:
```suggestion
# 01. GET /v2/registrations with 'limit' == 0 and options=count and see no registrations and fiware-total-count: 0
# 02. POST /v2/registrations to create registration
# 03. POST /v2/registrations to create another registration
# 04. GET /v2/registrations with 'limit' == 0 and options=count and see no registrations and fiware-total-count: 2
# 05. GET /v2/registrations with 'limit' == 0 and see no registration
```

Same changes in echo statements and expectations in the .test file.

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3994/commits/cf68c96ee05fa0fd7b53156f8e2b86761a386d0f